deploy/stacks/container.py

```diff
@@ -49,48 +49,88 @@ def __init__(
         self.task_role = self.create_task_role(envname, resource_prefix, pivot_role_name)
         self.cicd_stacks_updater_role = self.create_cicd_stacks_updater_role(envname, resource_prefix, tooling_account_id)
 
-        cdkproxy_task_definition = ecs.FargateTaskDefinition(
+        cdkproxy_container_name = f'container'
+        cdkproxy_log_group = self.create_log_group(
```

Comment:
The logging has not been added to the task definition

Reply:
Ok, I see that we created `LogConfigurationProperty` . Then we should removed this log group as it's not used

Reply:
The `LogConfigurationProperty` in its current form doesn't create the log group itself, it is just a parametrization, which tells which already existing log group should be used. The creation of the log group itself is happening here via `self.create_log_group`.

I checked it, and according to [this documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html), you could specify within the options of the `LogConfigurationProperty` the following: "awslogs-create-group": "true"
There are 2 problems with this:
1. I tried deploying it, and it doesn't work. No errors, it's visible in the JSON description of the ECS task definition, but the log group isn't created within Cloudwatch.
2. Even if it would work, you can't specify the 2 further attributes which were defined [here](https://github.com/awslabs/aws-dataall/blob/main/deploy/stacks/container.py#L479) via the already existing method.

Because of this, I'd keep the log group creation as it is done currently.

Reply:
Got it, thanks for the clarification! I was confused when I found that `cdkproxy_log_group` created, but it wasn't used. I thought that we should have deleted as it seemed like a leftover to me.

Then I have a minor suggestion which we could implement to avoid the confusion. I see that `LogConfigurationProperty` and `LogGroup` are implicitly bound by log_group_name which is the same.  We could make it more explicit by:
```
log_configuration=ecs.CfnTaskDefinition.LogConfigurationProperty(
                    log_driver="awslogs",
                    options={
                        "awslogs-group": cdkproxy_log_group.log_group_name

```
It should emphasize that there is a connection between those two. 
What do you think? :)